### ISSUE TITLE
ng2-select.css file is missing from the npm pacakge

### ISSUE THREAD

**Issue by danrevah** [valor-software/ng2-select#571]
After installing `npm install ng2-select` there's no css file inside the package.
Shouldn't there be ?

**Comment by SergeyKuryatnick** [comment]
yep, this file is not downloaded, because it used only for demo.

**Comment by srhlv** [comment]
Confused me too, since it's in quick start steps

**Comment by ermarkar** [comment]
Same issue but I copied the ng2-select.css file from the demo/app/assets/css folder. But still UI for drop down is not good. Am I still missing something? I am not using bootstrap but angular material 2.

![image](https://cloud.githubusercontent.com/assets/10824292/23057023/4135dc06-f514-11e6-901b-f1e9ce1d8977.png)

**Comment by shprink** [comment]
I used https://github.com/basvandenberg/angular2-select, it is well explained.

**Comment by manhcuong06** [comment]
Thanks @shprink  very much :).

**Comment by jrotondo-madaket** [comment]
Am am having the same CSS problem as @ermarkar above (component works, but styles are busted). I've followed the directions in @shprink's link, but there really wasn't a lot of detail on where to install the CSS, JS and font assets, and then how to link them. Can someone provide more specific instructions? Thanks to anyone who can help.

**Comment by FatmaBenMoussa** [comment]
Thank you @shprink

**Comment by suparnavg** [comment]
Hi Valor team, thank you so much for providing us with these excellent tools.

I have the same question: How do we edit the styles of the component? The ng2-select.css file is missing in the module. How can it be acquired, or are there any other workarounds that we can use?

Specifically, I need to reduce the height of the select input.

Thank you!

**Comment by nishant-befree** [comment]
Thank you so much  @shprink :).
